build.sbt

```diff
@@ -5,17 +5,11 @@ organization := "com.typesafe.sbt"
 name := "sbt-aspectj"
 version := "0.10-SNAPSHOT"
 
-libraryDependencies += "org.aspectj" % "aspectjtools" % "1.8.9"
+libraryDependencies += "org.aspectj" % "aspectjtools" % "1.8.10"
 
 publishMavenStyle := false
 
 bintrayOrganization := Some("sbt")
 bintrayRepository := "sbt-plugin-releases"
 bintrayPackage := name.value
 bintrayReleaseOnPublish := false
-
-crossBuildingSettings
-CrossBuilding.crossSbtVersions := Seq("0.12", "0.13")
-CrossBuilding.scriptedSettings
```

Comment:
This needs to switch from the `sbt-cross-building` scriptedSettings to the regular scripted plugin and settings.